cmd/starbridge/transform/transformTx.go

```diff
@@ -8,36 +8,38 @@ import (
 	"github.com/stellar/starbridge/cmd/starbridge/model"
 )
 
-// TODO this destination needs to be input from the input transaction on the remote chain from the memo or similar
-var destinationAccountForStellar = "GBUKJ5TXOBDB5SKVCEV27OYLHUL2ZD3OITMVFNT7MZ4LPO3EECTLADCU" // var destinationSecretKey = "SB6HKIGE6KKAOMSFY7G7VFIKNKMSQ7QRASFI65CVJLTZ7SUPLY4FAZ3L"
-
 // MapTxToChain converts a given transaction to the destination chain along with mapping the assets appropriately to the right contracts
-func MapTxToChain(tx *model.Transaction, destinationChain *model.Chain) (*model.Transaction, error) {
-	if destinationChain != model.ChainStellar {
+func MapTxToChain(tx *model.Transaction) (*model.Transaction, error) {
```

Comment:
our mapping logic has been updated, particularly since we updated the datamodel of our `model.Transaction` to explicitly save the Ethereum event data as a separate item so it is more correct.

now we use the data from the Ethereum Event to set up the converted Stellar transaction.